nuxeo-features/nuxeo-automation/nuxeo-automation-features/src/main/java/org/nuxeo/ecm/automation/core/operations/services/bulk/BulkRunAction.java

```diff
@@ -50,7 +50,7 @@
  *
  * @since 10.2
  */
-@Operation(id = BulkRunAction.ID, category = Constants.CAT_SERVICES, label = "Run a bulk command", addToStudio = true, description = "Run a bulk action on a set of documents expressed by a NXQL.")
+@Operation(id = BulkRunAction.ID, category = Constants.CAT_SERVICES, label = "Run a bulk command", description = "Run a bulk action on a set of documents expressed by a NXQL.", addToStudio = false)
```

Comment:
Don't we keep it in order to run async operation (such as event listener) on BAF? [NXP-26138](https://jira.nuxeo.com/browse/NXP-26138)

Reply:
Maybe indeed, i'm waiting for confirmation and run the T&P in the mean time to gain time. I'll request a review from @bchauvin and @aescaffre, thanks

Reply:
@aescaffre is supposed to discuss that one with core team to see how we can provide a map parameter from Studio. This prevents using the operation from Studio in my opinion.

Reply:
[NXP-26138](https://jira.nuxeo.com/browse/NXP-26138) fixed this Map parameter by replacing it by a simple `String`. AFAIK this could be used in Studio now.

Reply:
Agreed, this solution allows to expose it now.

Reply:
👍